recipes/spacy-lookups-data/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "spacy-lookups-data" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spacy_lookups_data-{{ version }}.tar.gz
+  sha256: 7831260fa75d9a61b3dc16c0853662285e8403ba714042435611277ea9b94ca5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - setuptools
+    - srsly >=0.1.0,<1.1.0
+    - pathlib ==1.0.1  # [py27]
```

Comment:
I don't think you need those dependencies to run the project, don't you think?
It seems that it is just a data repo. So, just ``python`` should be enough. I think those dependencies are necessary to test the project, but not to run it. Am I right?

Reply:
setuptools is necessary, but you're right the others aren't:

https://github.com/explosion/spacy-lookups-data/blob/master/spacy_lookups_data/__init__.py